target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -80,6 +80,11 @@ comfast,cf-e560ac)
 	ucidef_set_led_switch "lan3" "LAN3" "$boardname:blue:lan3" "switch0" "0x08"
 	ucidef_set_led_switch "lan4" "LAN4" "$boardname:blue:lan4" "switch0" "0x10"
 	;;
+comfast,cf-ew72)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:blue:lan" "eth0"
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:blue:wan" "eth1"
```

Comment:
You will have to use ucidef_set_led_switch for one of the interface to properly indicate connection state.

Reply:
I'm afraid I don't understand.  Something like the following:
ucidef_set_led_switch "lan" "LAN" "$boardname:blue:lan" "switch0" "0x02"

Sorry, this is my first new device so I'm still new to devicetree and LED configurations.

Reply:
lan and wan port are connected differently internally, but so only one can properly show connection state with ucidef_set_led_netdev. The other one will need something like you have provided, only with the correct value for the last parameter...

Reply:
Where would "switch0" and "0x02" be defined so I know what values to use?

Reply:
I just confirmed there doesn't appear to be any internal switches on this device.  Additionally, this current configuration works... the LAN port is lit up when the cable is plugged in and blinks with activity, when unplugged the LED turns off (both the LAN and WAN port support PoE).  When the WAN port is plugged in it turns on and blinks with activity, and when unplugged it turns off.  When both plugged in both work (activity) independent of each other.  Unfortunately it's impossible to test with both unplugged.

Additionally, it seems this is how the Comfast CF-E314N-v2 is configured in the same LED script.

I'm going to mark this as "resolved", but if I do need to revisit this please let me know.  Thank you.

Reply:
Both comfast,cf-e313ac and comfast,cf-e5 share your setup, and they use `ucidef_set_led_switch "lan" "LAN" "$boardname:blue:lan" "switch0" "0x02"`. If I have time tomorrow, I'll try to look for the explanation ...

Reply:
Okay, I'll try that and see what happens...

Reply:
Changing:
ucidef_set_led_netdev "lan" "LAN" "$boardname:blue:lan" "eth0"
To:
ucidef_set_led_switch "lan" "LAN" "$boardname:blue:lan" "switch0" "0x02"
also seems to work.  The blink pattern is noticably different, but still works more or less as expected (blinks with activity, but more rhythmic/slower).  I think the behavior with the netdev line is preferable, but since this works too I'm okay with it if it conforms to the code guidelines.